### DISCUSSION TITLE
Unreviewed pull requests

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/docs#79] [General]
@macmarrum I have found 4 pull requests which have been waiting quite a long time.

It becomes important to merge them before changing the directory structure.

Because nobody has reviewed them since 4 weeks I would like to install a bot automatically merging all pull requests not reviewed within 7 days and not having wip in their title. It would give you the opportunity to check any PRs before merging as you wanted.

Before the bot is created I would merge such PRs myself. Having no time to do reviews should IMHO not be a blocker.

How do you see it?

**Comment by macmarrum** [comment]
@dpolivaev 

Here's an excerpt from https://github.com/freeplane/docs/discussions/62, which explains how I see the merging activity

>> -  Who is responsible for merging the PRs?

> I know that many projects grant collaboration rights only after people have proven to be committed (measured by regular or significant contributions). Some projects require a PR review and approval of at least 1 "trusted user" before merging can be done. In any case, by creating a PR, the submitter proposes changes and promises to see them through until the PR is closed. Therefore the submitter is ultimately responsible for merging; if no collaboration rights, then for chasing a trusted user to merge it for them. Having said that, I am fine with other people merging, as long as the changed content's author's right to comment is honored and all parties agree to go ahead

From you post, I understand that you'd like a bot to be responsible for merging PRs.
Let's keep in mind that a person might react with a comment on the 7th day, and the discussion might continue for some more days.
If your bot can recognize that a PR is still being discussed and wait for the participants to finish, then it's fine with me.

As far as the four PRs are concerned
- https://github.com/freeplane/docs/pull/69
- https://github.com/freeplane/docs/pull/73
- https://github.com/freeplane/docs/pull/75
- https://github.com/freeplane/docs/pull/77

I have not been mentioned/asked to review them (in particular, no content of mine was changed), so I am fine with having them merged.

**Comment by sifran-github** [comment]
I personally think that good change that is not merged is a better situation than a bad change that is merged, so i think that a manual overview is better.
but i didn't really go over these PRs so its just my humble opinion.
